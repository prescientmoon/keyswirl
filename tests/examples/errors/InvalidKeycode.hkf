module exporting ()

alias foo = "wrong"
