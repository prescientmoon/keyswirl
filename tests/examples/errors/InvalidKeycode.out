[error InvalidKeycode]: Invalid keycode
     +--> errors.InvalidKeycode@3:13-3:20
     |
   3 | alias foo = "wrong"
     :       ^---  ^------
     :       |     `- "wrong" is not a valid keycode!
     :       `- while checking this alias
     :
     | Hint: For a list of all the available keycodes, check ...
-----+
